modules/mono/editor/bindings_generator.cpp

```diff
@@ -1731,6 +1731,41 @@ Error BindingsGenerator::_generate_cs_type(const TypeInterface &itype, const Str
 		output << INDENT1 "}\n";
 	}
 
+	//Generate StringName for all class members
+	StringBuilder property_names;
+	StringBuilder method_names;
+	StringBuilder signal_names;
```

Comment:
Now that all this code is together here, these builders are not needed anymore. You can move each each `public new class` and the closing braces declaration next to each loop.